docs/index.rst

```diff
@@ -37,6 +37,7 @@ the :setting:`CONSTANCE_CONFIG` section, like this:
         'django.contrib.messages',
         ...
         'constance',
+        'constance.backends.database',
```

Comment:
This will make it harder to use/understand. What happened with the backend setting?

Reply:
This application was added to INSTALLED_APPS for migrations consistency. 
If I remove it then migration to new app would not be transparent for users of this battery(and it can break there own migrations as Mogost mentioned)
The way how it is made now version updating is transparent.

Nothing has happened with backend setting. It works as usual.

Do you have any thought how to make it better?

Reply:
@Mogost how do you think we can get the same result without adding this extra line?

Reply:
I am not sure. May be `replaces` option in `constance` app will help.

Reply:
What is **replaces** option?

Reply:
It's what django uses on squashmigrations https://docs.djangoproject.com/en/4.0/topics/migrations/#squashing-migrations
After squashing you will have replace the attribute in migration.

Reply:
```
    replaces = [
        ("migrations", "0001_initial"),
        ("migrations", "0002_second"),
    ]

```

Reply:
I have checked this variant. Squashing of migrations is only possible within one application [only possible within one application ](https://docs.djangoproject.com/en/4.0/ref/django-admin/#django-admin-squashmigrations).
In our case there are 2 applications: database & constance.

Reply:
@Mogost any thoughts?

Reply:
Nothing from my side.
I don't know how to do it better.
May be we can forget about my point [here](https://github.com/jazzband/django-constance/pull/469?notification_referrer_id=NT_kwDOAAsEK7EyOTMzNDU4MjQ4OjcyMTk2Mw#issuecomment-1040508408)
> Just for example, I found a place in my project where my migrations were dependent on Constance migrations like that:
> ```python
> class Migration(migrations.Migration):
>     dependencies = [
>        ('database', '0001_initial'),
>    .....
>```

We can forget about this and bump the major version with a few words in release notes.

Reply:
@camilonova  It seems that we already have `constance.backends.database` in our [docs here](https://django-constance.readthedocs.io/en/latest/backends.html#database). So it turns out it's always been there.

This was probably done with the assumption that migration is unnecessary if the data is stored in redis and not in the database.

@jezdez is the original author of this code. And we probably need his advice here. I have just mailed him.
We are now actually changing the functionality that could lead to data loss for our users. We need to be careful with this pr.

Reply:
I agree. At this point, I'm not in favor of making people add two lines at the INSTALLED_APPS setting to use this library.

Reply:
Finally I merge develop, fix conflicts and you will be able to merge MR?